src/openforms/forms/admin/form.py

```diff
@@ -70,16 +69,18 @@ def render_change_form(
         context.update({"use_react": self.use_react(request)})
         return super().render_change_form(request, context, add, change, form_url, obj)
 
+    def changelist_view(self, request, extra_context=None):
+        context = {
+            "has_change_permission": self.has_change_permission(request),
+        }
+        context.update(extra_context or {})
+        return super().changelist_view(request, context)
```

Comment:
Why is this needed?

Reply:
This was part of the `VersionAdmin` https://github.com/etianen/django-reversion/blob/master/reversion/admin.py#L229, and without it the `import form` button in the admin is no longer visible.